distribution/sender/routes/active-mq-publisher.xml

```diff
@@ -0,0 +1,132 @@
+<routes xmlns="http://camel.apache.org/schema/spring">
+
+    <route id="active-mq-publisher" errorHandlerRef="outBoundErrorHandler">
+    
+        <!-- TODO(luis.pinto) Make the delay configurable -->
+        <from uri="scheduler:active_mq_publisher?initialDelay=15000&amp;delay=60000" />
+
+        <log message="Fetching next 1000 sender sync messages" loggingLevel="DEBUG" />
+
+        <toD uri="jpa:SenderSyncMessage?query=SELECT m FROM SenderSyncMessage m WHERE m.sent = false ORDER BY m.dateCreated ASC, m.id ASC&amp;maximumResults=1000" />
```

Comment:
With the number of processor routes increasing, may be we should reduce the limit to 500 to cut on the memory footprint

Reply:
Changed. Configuration created.